types/hour.pp

```diff
@@ -2,8 +2,8 @@
 type Cron::Hour = Variant[
   Integer[0,23],
   Pattern[/(?x)\A(
-    \* ( \/ ( 1?[0-9]|2[0-3]|[3-9] ) )?
-    |       ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[3-9] ) )? )?
-        ( , ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[3-9] ) )? )? )*
+    \* ( \/ ( 1[0-9]|2[0-3]|[1-9] ) )?
+    |       ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[1-9] ) )? )?
+        ( , ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[1-9] ) )? )? )*
```

Comment:
:+1: for rejecting `*/0`.

On a side note, anything greater than `/12` does not really make sense is is likely an error.  Maybe we do not want to accept such values?